src/DependencyInjection/Configuration.php

```diff
@@ -117,10 +117,10 @@ public function getConfigTreeBuilder()
             ->end()
             ->scalarNode('slugify_service')
                 // NEXT_MAJOR: reword this info message
-                ->info('You should use: sonata.core.slugify.cocur, but for BC we keep \'sonata.core.slugify.native\' as default')
+                ->info('You should use: sonata.page.slugify.cocur, but for BC we keep \'sonata.core.slugify.native\' as default')
 
-                // NEXT_MAJOR: use "sonata.core.slugify.cocur" instead of "sonata.core.slugify.native" as default value
-                ->defaultValue('sonata.core.slugify.native')
```

Comment:
Changing the default value would be a bc break, maybe register an alias to the new value on services?